### PR TITLE
feat(finproms): added info for investment risk

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Comment by sstephanou-bc** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [b0814c81\-56a5\-46b2\-8632\-3f03118440ce](https://eu.ast.checkmarx.net/projects/f3c879d8-da6e-4aaa-aeee-f377cc306d79/scans?id=b0814c81-56a5-46b2-8632-3f03118440ce&branch=feat%2FGROWUX-3484-fin-prom-message)
### No New Or Fixed Issues Found

File: packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx [hunk 1]
```diff
@@ -319,6 +321,7 @@ class TransactionsContainer extends React.PureComponent<Props> {
                 sourceType={sourceType}
               />
             ))}
+          {isCustodialCoin && <RiskInvestment coin={coin} />}
```
Comment (dkremniov-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx]: Let's check with Seba if we actually need to check for whether coin is custodial or not. Separate between DeFi and Custodial is clear in superapp but not in v4. So maybe we should be showing the message to UK users all the time on this page.

Also `isCustodialCoin` does not seem to include `BTC`.

Comment (dkremniov-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx]: Also this component has to be shown only to UK users.

Comment (Tangee69) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx]: I don't understand

File: packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/Lines.styles.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  ${({ isCollapsed }) =>
+    isCollapsed
+      ? css`
+          max-height: 40px;
+        `
+      : css`
+          max-height: 200px;
+        `}
+`
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/Lines.styles.ts:10]: A personal opinion here - why not `max-height: ${(props) => props.isCollapsed ? '40px' : '200px'}` ?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/Lines.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+export const Lines = ({ lines }: Props) => {
+  const [isCollapsed, setIsCollapsed] = useState(true)
+
+  const toggleCollapse = () => {
+    setIsCollapsed(!isCollapsed)
+  }
+
+  return (
+    <div>
+      <ListContainer isCollapsed={isCollapsed}>
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/Lines.tsx]: I'm not sure if it was in v4 or v5, but linter didn't like props with camelCase. If it's the case, prepend a `$` it to make it a [transient prop](https://styled-components.com/docs/api#transient-props)

File: packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/index.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import data from './data.json'
+import { Lines } from './Lines'
+import { Arrow, ContentWrapper, MainWrapper, Title } from './RiskInvestment.styles'
+
+type Props = {
+  coin: AllCoinsType
+}
+
+export const RiskInvestment = (props: Props) => {
+  const { coin } = props
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/index.tsx]: Could be destructured directly in the definition if it's the only one

**Review by dkremniov-bc — APPROVED** [review]

**Pull request merged**